eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/AnswerProperties/Properties/DateFormat.js

```diff
@@ -1,15 +1,27 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { enableOn } from "utils/featureFlags";
 
 import { Field, Select } from "components/Forms";
 
 const DateFormat = ({ id, value, onChange }) => (
   <Field>
-    <Select value={value} onChange={onChange} id={id} name={id}>
-      <option value="dd/mm/yyyy">dd / Month / yyyy</option>
-      <option value="mm/yyyy">Month / yyyy</option>
-      <option value="yyyy">yyyy</option>
-    </Select>
+      {enableOn(["hub"]) ? 
+
+        <Select data-test="select" value={value} onChange={onChange} id={id} name={id}>
+            <option data-test="day-month-year" value="dd/mm/yyyy">dd / mm / yyyy</option>
+            <option data-test="month-year" value="mm/yyyy">mm / yyyy</option>
+            <option data-test="year" value="yyyy">yyyy</option> 
+        </Select>
+
+      : 
+        <Select data-test="select" value={value} onChange={onChange} id={id} name={id}>
+          <option data-test="day-month-year" value="dd/mm/yyyy">dd / Month / yyyy</option>
+          <option data-test="month-year" value="mm/yyyy">Month / yyyy</option>
+          <option data-test="year" value="yyyy">yyyy</option>
+        </Select>
+      }
+
```

Comment:
```suggestion
      <Select data-test="select" value={value} onChange={onChange} id={id} name={id}>
          <option data-test="day-month-year" value="dd/mm/yyyy">dd / {enableOn(["hub"]) ? "Month" : "mm"} / yyyy</option>
          <option data-test="month-year" value="mm/yyyy">{enableOn(["hub"]) ? "Month" : "mm"} / yyyy</option>
          <option data-test="year" value="yyyy">yyyy</option> 
      </Select>
```

Does this work? (or better - move out into e.g.

```javascript
const monthText = enableOn(["hub]) ? "Month" : "mm";
```

and use that instead

Reply:
Ah great shout. I should've picked that up on my refinement pass.

Reply:
That's been changed.